### ISSUE TITLE
Page listing all core libraries

### ISSUE THREAD

**Issue by hdgarrood** [purescript/pursuit#326]
I think it could be helpful to put all the core libraries on their own page on Pursuit, so that people who are relatively new to PureScript can look through it and familiarise themselves with them. I think this would also be a good place to clarify what a "core library" is, as this doesn't seem to be particularly clear to newcomers right now. I'd want to replace the link to Prelude on the front page with a link to this core libraries page (of course, Prelude would then be listed on that page).

I think this would go a long way toward addressing the issue raised in #158 and also some of the issues raised in https://github.com/purescript/documentation/issues/43.

**Comment by Thimoteus** [comment]
What I've wanted for a while is for pursuit to order first by core libraries, then third party libraries. A lot of names are used across libs and I almost always want the one that defined in a core repo.

**Comment by hdgarrood** [comment]
Do you mean when searching by name?

**Comment by Thimoteus** [comment]
Yes, for example: [apply](https://pursuit.purescript.org/search?q=apply), it's not until the 4th listing that I get the definition of `<*>`

**Comment by hdgarrood** [comment]
Right, agreed. I'd prefer to consider that in a separate issue though.

**Comment by Thimoteus** [comment]
Understood.
